container/common/container_hints.go

```diff
@@ -25,7 +25,7 @@ import (
 	"os"
 )
 
-var argContainerHints = flag.String("container_hints", "/etc/cadvisor/container_hints.json", "location of the container hints file")
+var ArgContainerHints = flag.String("container_hints", "/etc/cadvisor/container_hints.json", "location of the container hints file")
```

Comment:
Got it. Ideally, we should push the flag up since you are introducing new *shiny and clean* code. But I'm fine with punting that to a separate PR.